src/cli/commandModules/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from "./check";
 export * from "./copy";
-export * from "./list/";
 export * from "./list";
+export * from "./listCommands";
+export * from "./listStates";
```

Comment:
Do you know if the list command is still working on the CLI?

The reason for the directory layout was that list.ts loads the subcommands subdirectory (just as the main cli loads the commandModules subdirectory). I think list.ts needs to be updated.

Reply:
The  list commands still work (just tested). All modules get loaded from the src/cli.ts `commandDir` call. I think the `commandDir` call in src/cli/commandModules/list did not do anything. I removed the builder method.

Reply:
The list command (`bluehawk list`) no longer lists the subcommands. This is what the commandDir call was doing.

Compare your change (above) with 0.6.0 (below):

![image](https://user-images.githubusercontent.com/20050130/134736205-77977da2-6cca-455e-9428-afbf6815ebb6.png)

Reply:
Sorry, I now realize I changed the API and made `states` and `commands` no longer subcommands of `list`. Thanks for helping me understand the mechanism by which subcommands are created over slack.

I think I implemented the fix, let me know if you have any additional concerns.